scripts/auto_mesh.py

```diff
@@ -0,0 +1,290 @@
+# Adds connectors in order to ensure a full mesh for a deployment or
+# similar
+
+from __future__ import print_function
```

Comment:
this is needed only in Python 2

Reply:
True. I will update these python scripts as well.